src/test/rpc/Catalogue_test.cpp

```diff
@@ -829,7 +831,7 @@ class Catalogue_test : public beast::unit_test::suite
         {
             auto result = env.client().invoke(
                 "catalogue_status", Json::objectValue)[jss::result];
-            std::cout << to_string(result) << "\n";
+            // std::cout << to_string(result) << "\n";
```

Comment:
just remove?

Reply:
I have made it so that you only need to comment it out during testing. I can delete it if you wish.

Reply:
It really doesn't bother me at all